ketcher-autotests/tests/utils/canvas/tools/selectNestedTool/selectNestedTool.ts

```diff
@@ -9,22 +9,25 @@ import { getToolType, openTool } from './helpers';
  *
  * @param page - playwright page object
  * @param toolElementId - tuple [presses: number, domElementId: string]
+ * @param currentToolId - current selectede-group tool
  * presses - specifies the number of times to press Tab key
  * domElementId - id of the element to search
  */
 export const selectNestedTool = async (
   page: Page,
   toolElementId: [presses: number, domElementId: string],
+  currentToolId?: [presses: number, domElementId: string],
```

Comment:
And could you, please, clarify what it is used for?

Reply:
When we create a r-group with fragments, we need to switch the r-group tool from ATTACHMENT_POINTS to R_GROUP_FRAGMENT. If we didnt pass in this ATTACHMENT_POINTS as a param, selectNestedTool would call openTool func to query the default element with testId of 'rgroup-label', while actually there is no such element at this point, only 'rgroup-attpoints' exist.
![image](https://github.com/epam/ketcher/assets/35161544/fac7c9e7-2269-44ba-91e1-143923a1ed6d)